Tools/boringNetworkTool/sending_config.json

```diff
@@ -0,0 +1,54 @@
+{
+  "config": {
+    "ip": "localhost",
+    "port": 8888,
+    "useTCP": false,
```

Comment:
I think it might be better if this is commsType, or something similar, and then it parses for TCP or UDP as the value. If you make the option to act as the tcp server, you could then have a setting called servePort which would determine that.